org.eclipse.mylyn-target/org.eclipse.mylyn.target.target

```diff
@@ -96,4 +96,5 @@
 			<unit id="org.objenesis" version="0.0.0"/>
 		</location>
 	</locations>
+	<targetJRE path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-21"/>
```

Comment:
I thought this line would fix the license checking because it helped with the Platform.  But apparently not.

Reply:
May be we need to pass a parameter to fulfill this https://github.com/eclipse-dash/dash-licenses/blob/bc40f27ca7543063a6fc99d97bc308dd0b568f9c/.github/workflows/mavenLicenseCheck.yml#L46

Reply:
Yes, that would make sense.

I'm not sure I understand the test failures either.  Maybe it's more of a Java 21 thing.

I don't have further time to help on this. I hope to leave it your guys' capable hands.  Okay?

Reply:
Unfortunately I have no idea how to add more commits to the PR, that was created from the fork of another person. 
Is this possible at all or should I open another PR instead?

Reply:
Don’t worry. Just close this and create a better pr. I won’t be the slightest bit offended!!

Reply:
Looks like you can do it with this: https://stackoverflow.com/questions/44030176/how-to-modify-someone-elses-github-pull-request